packages/instrumenter/src/syntax/index.ts

```diff
@@ -4,18 +4,19 @@ export enum AstFormat {
   Html = 'html',
   JS = 'js',
   TS = 'ts',
-  TSX = 'tsx',
+  Tsx = 'tsx',
```

Comment:
I've implemented `Tsx` as its own `AstFormat`. That way it's a bit more explicit.

Reply:
Also, chose PascalCase, because that's in line with other casing within our repo (i.e. `Html`)